config/roles.yml

```diff
@@ -255,3 +255,26 @@ point_in_time_full_access:
         - '*'
       allowed_actions:
         - 'manage_point_in_time'
+
+# Allows users to see security analytics detectors and others
+detectors_read_access:
+  reserved: true
+  cluster_permissions:
+    - 'cluster:admin/opendistro/securityanalytics/alerts/get'
+    - 'cluster:admin/opendistro/securityanalytics/mapping/get'
+    - 'cluster:admin/opendistro/securityanalytics/mapping/view/get'
+    - 'cluster:admin/opendistro/securityanalytics/findings/get'
+    - 'cluster:admin/opendistro/securityanalytics/rule/get'
+    - 'cluster:admin/opendistro/securityanalytics/detector/get'
+    - 'cluster:admin/opendistro/securityanalytics/rule/search'
+    - 'cluster:admin/opendistro/securityanalytics/detector/search'
+
+# Allows users to use all security analytics functionality
+detectors_full_access:
+  reserved: true
+  cluster_permissions:
+    - 'cluster:admin/opendistro/securityanalytics/mapping/*'
```

Comment:
Please sort all permissions in alphabetical order to make them easier to compare between read/full roles

Reply:
done